### DISCUSSION TITLE
Self-Learning is showing in progress even if it is completed

### DISCUSSION THREAD

**Discussion by priyanshu688** [github/india#561] [General]
Can anyone help me out with this

**Comment by SudarshanZimal** [comment]
Same problem .

Please check, I have completed all self learning modules but currently show in progress.
My  ms learn profile.
https://docs.microsoft.com/en-us/users/sudarshanzimal-8599/achievements

**Comment by Rahul4343** [comment]
Even i am suffering from this 
![image](https://user-images.githubusercontent.com/52530029/152169184-0ea90ace-6130-4738-ad9b-77b5acc43278.png)

**Comment by sagarkulkarni63** [comment]
Even I'm facing same issue, wrote multiple emails to them regarding this but no proper response. Just replying with generic solutions which aren't useful.

**Comment by shri3016** [comment]
Hey,Is your profile showing that the courses are completed now?

**Reply by sagarkulkarni63** [reply]
Yes

**Reply by shri3016** [reply]
In how many days was it updated?

**Reply by sagarkulkarni63** [reply]
Maybe 2-3 days. Out of 5 mandatory ones only 2 updated.

**Reply by Rahul4343** [reply]
Same here only few are updated.
![image](https://user-images.githubusercontent.com/52530029/152742943-897b23e4-4ed6-445e-8da4-4047da9d9d5d.png)
